utils/pose_detector.py

```
@@ -1,12 +1,14 @@
 """
 Pose Detector module using TensorFlow's MoveNet for skeletal tracking
 """
+
 import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)